src/modules/auth/components/EnterPasswordForm/enterPasswordForm.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import WalletVisual from '@wallet/components/walletVisual';
+import useDecryptionAccount from '@account/hooks/useDecryptionAccount';
+import { Input } from 'src/theme';
+import Box from 'src/theme/box';
+import BoxContent from 'src/theme/box/content';
+import { PrimaryButton } from 'src/theme/buttons';
+import styles from './enterPasswordForm.css';
+
+const EnterPasswordForm = ({ accountSchema, onEnterPasswordSuccess }) => {
+  const { t } = useTranslation();
+  const [password, setPassword] = useState('');
+  const [feedbackError, setFeedbackError] = useState('');
+
+  const onSubmit = () => {
+    const { privateToken, recoveryPhrase, error } = useDecryptionAccount(accountSchema, password);
+    if (error) {
+      return setFeedbackError(error);
+    }
+
+    return onEnterPasswordSuccess({ privateToken, recoveryPhrase });
+  };
+
+  return (
+    <Box className={styles.container}>
+      <BoxContent className={styles.content}>
+        <h1>{t('Enter your password')}</h1>
+        <p className={styles.subheader}>
+          {t('Please provide your device password to backup the recovery phrase.')}
+        </p>
+        <WalletVisual
+          className={styles.avatar}
+          address={accountSchema?.metadata?.address}
+        />
+        <p className={styles.accountName}>{accountSchema?.metadata?.name}</p>
```

Comment:
name is optional 
{accountSchema?.metadata?.name &&  <p className={styles.accountName}>{accountSchema?.metadata?.name}</p>}